crates/sqlite-db/src/rollover/load.rs

```diff
@@ -145,7 +145,9 @@ async fn load_revoked_commit_transactions(
             revocation_sk_ours: row
                 .revocation_sk_ours
                 .map(|revocation_sk_ours| revocation_sk_ours.into()),
-            revocation_sk_theirs: row.revocation_sk_theirs.into(),
+            revocation_sk_theirs: row
+                .revocation_sk_theirs
+                .map(|revocation_sk_theirs| revocation_sk_theirs.into()),
```

Comment:
❓ What does that do?

Reply:
Before, the `revocation_sk_theirs` was declared as `NOT NULL`, so we would always get a `sqlite_db::models::SecretKey` when loading from the database. We would then convert it into a `secp256k1::SecretKey` with the `.into()`.

With this PR, the `revocation_sk_theirs` is declared as `NULL`. This means that we load an `Option<sqlite_db::models::SecretKey>` from the database. We now need to convert it into an `Option<secp256k1::SecretKey`. To do so we map the contents of the `Option` with the appropriate `Into` (auto-generated by implementing `From`) implementation.

You can even write:

```rust
.map(secp256k1::SecretKey::from)
```